src/go/rpk/pkg/cli/cmd/common/common_test.go

```diff
@@ -263,10 +266,29 @@ func TestKafkaAuthConfig(t *testing.T) {
 		password:  "contraseño",
 		mechanism: "SCRAM-SHA-512",
 		expected:  &config.SCRAM{User: "usuario", Password: "contraseño", Type: "SCRAM-SHA-512"},
+	}, {
+		name: "it should pick up the values from env vars if the vars' values is empty",
```

Comment:
could you also add test that it won't pick up the value from env vars is flag is specified?

Reply:
Good idea!